fdbcli/TenantCommands.actor.cpp

```diff
@@ -366,7 +366,7 @@ ACTOR Future<bool> tenantListCommand(Reference<IDatabase> db, std::vector<String
 			state std::vector<TenantName> tenantNames;
 			if (clusterType == ClusterType::METACLUSTER_MANAGEMENT) {
 				std::vector<std::pair<TenantName, TenantMapEntry>> tenants =
-				    wait(MetaclusterAPI::listTenants(db, beginTenant, endTenant, limit, offset, filters));
+				    wait(MetaclusterAPI::listTenantMetadata(db, beginTenant, endTenant, limit, offset, filters));
```

Comment:
Ideally we could avoid doing the point lookups unless required (e.g. because we used filters). Possibly we could make that decision here, or we could also push the logic for that down into `listTenant()`.